x-pack/plugins/security_solution/common/search_strategy/security_solution/users/index.ts

```diff
@@ -11,6 +11,14 @@ export enum UsersQueries {
   details = 'userDetails',
   kpiTotalUsers = 'usersKpiTotalUsers',
   users = 'allUsers',
+  authentications = 'authentications',
+  authenticationsEntities = 'authenticationsEntities',
 }
 
 export type UserskKpiStrategyResponse = Omit<TotalUsersKpiStrategyResponse, 'rawResponse'>;
+
+export * from './all';
+export * from './common';
+export * from './kpi';
+export * from './details';
+export * from './authentications';
```

Comment:
Nit: we usually place the `*` direct exports at the top, after imports, and before type definitions.